### DISCUSSION TITLE
Reolink Camera Plugin Ony Using Sub Stream

### DISCUSSION THREAD

**Discussion by jeremyward** [koush/scrypted#1435] [Q&A] [answered]
Is there any way to configure the Reolink Camera Plugin to just harvest the low resolution sub stream and NOT the main stream?  I have a bunch of the RLC-810A and RLC-811A cameras.  I would like the Reolink NVR to record in 4k in case I need that.  I would also like to use Scrypted.  I know if the cameras stream 4k, Reolink uses h265 on that stream which is incompatible with Scrypted.  Rather than degrading the main stream, can I just point the Scrypted Plugin at ONLY the lower resolution sub stream and not suffer the loss of all the 4k data on the Reolink NVR?  I prefer not to use RTSP with or without email notifications.

**Comment by koush** [comment, marked as answer]
look in the streams tab.
